frontend/src/pages/settings/IndustryList.tsx

```
@@ -23,146 +28,336 @@ export const IndustryList: React.FC<IndustryListProps> = ({
+                  <button
+                    onClick={() => setRowsPerPage(rows)}
+                    className={`px-3 py-1 text-sm rounded transition-colors ${
+                      rowsPerPage === rows
+                        ? 'text-white border-2'
+                        : 'text-gray-400 hover:text-gray-300'
+                    }`}
+                    style={{
+                      backgroundColor: rowsPerPage === rows ? '#2E3333' : 'transparent',
+                      borderColor: rowsPerPage === rows ? '#66D9EF' : 'transparent',
+                    }}
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)